understanding-logic/concepts/modelet.rst

```diff
@@ -1,106 +1,114 @@
 Types
 =====
 
-.. _modelet:
+
 
 **modelet** ``$tType``
 ----------------------
 
-Models a `phenomenon <phenomenon.rst>`__ as the *thought* in the triangle
-of meaning. All modelets describe an instance of a
-`concept <concept.rst>`__, have an `origin <origin.rst>`__, and are
-modeled in a specific `formalism <formalism.rst>`__.
+Models a `phenomenon <phenomenon.html>`_ as the *thought* in the triangle of meaning. All modelets describe an instance of a `concept <concept.html>`_, have an `origin <origin.html>`_, and are modeled in a specific `formalism <formalism.html>`_.
+
+-  `Concept <concept.html>`_: the *phenomenon* of the
+   `modelet <modelet.html>`_, an instance of a class in the world model.
+-  `Origin <origin.html>`_: the agent’s perspective on the *phenomenon*
+-  `Formalism <formalism.html>`_: the representation of the *phenomenon*
 
--  `Concept <concept.rst>`__: the *phenomenon* of the
-   `modelet <#modelet>`__, an instance of a class in the world model.
--  `Origin <origin.rst>`__: the agent’s perspective on the *phenomenon*
--  `Formalism <formalism.rst>`__: the representation of the *phenomenon*
-   ###### Source: ``engines-and-modelets.tff``
+Source: ``engines-and-modelets.tff``
 
```

Comment:
Please add some information here about `empty_ms`, and `parents_of_modelet` since it looks like we are going the understanding state route.